src/Microsoft.DocAsCode.Build.Engine/PostbuildPhaseHandler.cs

```diff
@@ -9,11 +9,14 @@ namespace Microsoft.DocAsCode.Build.Engine
 
     internal class PostbuildPhaseHandler : IPhaseHandler
     {
-        private DocumentBuildContext _context;
+        public DocumentBuildContext Context { get; }
```

Comment:
i should make all of them public

---
In reply to: [89707591](https://github.com/dotnet/docfx/pull/929#discussion_r89707591) [](ancestors = 89707591)